kirara_ai/ioc/container.py

```diff
@@ -6,16 +6,29 @@
 # 使用 contextvars 实现线程和异步安全的上下文管理
 current_container = contextvars.ContextVar("current_container", default=None)
 
-
 class DependencyContainer:
     """
-    作用域容器，提供依赖注册和解析的核心功能。你可以在此获取一些系统的对象
+    作用域容器，提供依赖注册和解析的核心功能。你可以在此获取一些系统的对象。
+    基本用法：
+    ```python
+    from kirara_ai.llm.llm_manager import LLMManager
+    from kirara_ai.ioc.container import DependencyContainer
+
+    # 作为插件开发者你应该新创建一个子作用域，与系统全局作用域分开。
+    # 子作用域相互隔离，父作用域为系统全局作用域。
+    container = DependencyContainer().scoped()
+    # 传入LLMManager的类型名，容器会自动查找这个对象的实例。
+    llm_manager = container.resolve(LLMManager)
+    llm_manager.get_llm()
+    ...
+    ```
+
+    docs: https://docs.python.org/zh-cn/3.13/library/contextvars.html#module-contextvars
```

Comment:
```suggestion
    依赖注入容器，提供注册和解析的功能。你可以在此获取一些全局的对象。
    基本用法：
    ```python
    # 1. 注册全局对象 - 通常在初始化时使用
    container.register(YourObj, your_obj_instance)

    # 2. 获取全局对象 - 在你的逻辑代码中使用
    your_obj_instance = container.resolve(YourObj)

    # 3. 销毁全局对象 - 通常在系统/插件销毁时使用
    container.destroy(YourObj)

    # 4. 创建作用域容器 - 作用域容器内注册的对象只在作用域内可被访问
    # 离开作用域的上下文后无法取到该对象

    # 全局容器注册对象
    container.register(KiraraObj, kirara_obj)

    with container.scoped() as scoped_container:
        # 注册作用域对象
        scoped_container.register(YourObj, your_obj_instance)
        # 获取作用域对象
        scoped_container.resolve(YourObj)
        # 作用域容器也可以获取到全局容器的对象
        container.has(KiraraObj) #  返回 True

        # 甚至还能再创建新的作用域容器

        with scoped_container.scoped() as another_scoped_container:
            another_scoped_container.has(YourObj) # True

    # 离开作用域上下文后无法获取到该对象
    container.has(YourObj) # 返回 False
    ```
```